Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -365,11 +365,16 @@ void MarkupAccumulator::startAppendingNode(const Node& node, Namespaces* namespa
         m_nodes->append(const_cast<Node&>(node));
 }
 
+static String nodeNamePreservingCase(const Element& element)
```

Comment:
I don't think this helper is necessary. We can just repeat `element.tagQName.toString()`.

Reply:
@rniwa - Done. Thanks for suggestion.